src/main/java/com/google/edith/GroceryDataReader.java

```diff
@@ -52,6 +51,11 @@ public DealItem readFile(String itemName) throws IOException {
           item.setPrice(record[storeDataStartColumn + 1]);
           item.setWeight(record[storeDataStartColumn + 2]);
           item.setComment(record[storeDataStartColumn + 3]);
+          if (!expirationTime.isEmpty()) {
+            item.setExpiration(expirationTime);
+          } else {
+            item.setExpiration("NO_EXPIRATION");
+          }
```

Comment:
Typically it's best to keep boolean checks positive in the conditional as they're logically easier to follow/read.